backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/DocumentationUnitService.java

```diff
@@ -478,9 +477,10 @@ public DocumentationUnit getByDocumentNumberWithUser(String documentNumber, Oidc
   private Slice<DocumentationUnitListItem> retrieveCurrentProcessStepUser(
       Slice<DocumentationUnitListItem> documentationUnitListItems, OidcUser oidcUser) {
 
-    Map<UUID, User> userIdMap =
-        userService.getUsers(oidcUser).stream()
-            .collect(Collectors.toMap(User::id, Function.identity()));
+    Map<UUID, User> userIdMap = new HashMap<>();
+    for (User user : userService.getUsers(oidcUser)) {
```

Comment:
I find the name of the of the method not good. we have now a method which returns a list of users for one user? I think the method returns other user of the same doc office as the user? My idea is getUsersAtTheSameDocOfficeAs(oidcUser)?

Reply:
Second one? With this solution you always override the value in the map, right? So the last entry wins.